### PR TITLE
Remaining tasks for Cloud5 upgrade wizard

### PR DESCRIPTION
**Pull request by vwallfahrer**
Depends on https://github.com/crowbar/barclamp-crowbar/pull/1372

### CODE REVIEW

**Comment by tboerger** [comment]
:+1:

**Comment by tboerger** [comment]
As this PR includes the changes from #1372 I think we can just merge only this one :)

**Comment by rhafer** [comment]
@vwallfahrer please rebase on current master.

Also squash the last two commits (https://github.com/crowbar/barclamp-crowbar/commit/f37d287883b9f1ca7f6ca828c48e096976188243 basically reverts https://github.com/crowbar/barclamp-crowbar/commit/6e7c97d15feffca40ded74fe8d406f00b68d5956) . Otherwise this looks good to me.

File: crowbar_framework/app/assets/javascripts/application.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ jQuery(document).ready(function($) {
     ]
   });
 
-  $('[data-blockui]').live('submit', function(event) {
+  $('[data-blockui]').live('click submit', function(event) {
```
Comment (jsuchome) [inline crowbar_framework/app/assets/javascripts/application.js:88]: Is this correct? Maybe I have something wrong (I only have part of the patch applied), but I'm getting blocking UI for every submit, e.g. when I'm editing barclamp proposal

**Comment by rhafer** [comment]
Based on this PR I've updated most of the UI text (as a start). Please see: https://github.com/rhafer/barclamp-crowbar/commit/cef2e984262e81b13046fbe89e5681407c61b561

**Comment by rsalevsky** [comment]
+1

**Comment by MaximilianMeister** [comment]
+1

File: crowbar_framework/app/assets/javascripts/application.js (hunk 1 above)
Comment (rhafer) [inline crowbar_framework/app/assets/javascripts/application.js:88]: Yeah, something is wrong here. I have the same now. I'll create a bug report.

**Pull request merged**